Cabal/Distribution/Compat/Parsing.hs

```diff
@@ -102,8 +108,7 @@ sepBy p sep = sepBy1 p sep <|> pure []
 -- | @sepBy1 p sep@ parses /one/ or more occurrences of @p@, separated
 -- by @sep@. Returns a list of values returned by @p@.
 sepBy1 :: Alternative m => m a -> m sep -> m [a]
-sepBy1 p sep = (:) <$> p <*> many (sep *> p)
```

Comment:
FWIW, we should probably justt remove `*By1` functions. I don't recall they are really used anywhere (or then the `toList` can be put there).

Doesn't need to be done, if you don't feel like that.

Reply:
Yep, looks like it's only a few instances.

Personally I feel like the `*By1` naming conventions is nicer than trailing with `NonEmpty`, but I've gone for with it for now.